app/routes/agenda-items/agenda-item.ts

```diff
@@ -13,7 +13,9 @@ export default class AgendaItemRoute extends Route {
   @service declare keywordStore: KeywordStoreService;
 
   async model(params: DetailParams) {
-    const agendaItem = await this.store.findRecord('agenda-item', params.id);
+    const agendaItem = await this.store.findRecord('agenda-item', params.id, {
+      include: 'handled-by',
+    });
```

Comment:
**Issue:** The include is not needed. We already load it line 53: `const agendaItemHandling = await agendaItem.handledBy;`
We decide to split all require in small calls, in order to improve source of problem and avoid on big 500 error when a property or a link is broken.   

```suggestion
   // To improve error isolation, the required data are loaded below in separate calls. 
    const agendaItem = await this.store.findRecord('agenda-item', params.id);
```

Reply:
Oh, I missed that. I saw a `handled-by` call in the network tab and assumed it was done by the template access. Thanks. I'm not sure I understand the reasonings behind the split, but I'll just change it for now.

Reply:
@Windvis Before, we used require, but when something was broken in the data (like a type error in besluit value), the server returned a 500 error. The whole page was empty, and it was difficult to isolate the source of the problem. Now each dependency is loaded separately. We render the page even when some data is broken, and it is easier to find which data is broken.

Reply:
(I force pushed the revert, so a reset is needed if you want to test again)